external-crates/move/crates/move-compiler/tests/linter/incorrect_unnecessary_while_loop.move

```diff
@@ -0,0 +1,54 @@
+module 0x42::loop_test {
```

Comment:
Please split this file as previously discussed

Reply:
refactored